src/LCT.APICommunications/Interfaces/IJfrogAqlApiCommunication.cs

```diff
@@ -29,5 +29,7 @@ public interface IJfrogAqlApiCommunication
         /// <param name="path">repoName</param>
         /// <returns>AqlResult</returns>
         Task<HttpResponseMessage> GetPackageInfo(string repoName, string packageName = null, string path = null);
+
```

Comment:
Please add method description in the similar lines of the GetPackageInfo method

Reply:
Added